src/main/java/carpet/script/language/DataStructures.java

```diff
@@ -58,11 +59,13 @@ else if (lv.size() == 2 && lv.get(1) instanceof ListValue)
             String hwat;
             if (lv.size() == 1)
             {
+                if(lv.get(0) instanceof NullValue) return new ListValue(new ArrayList<>());
```

Comment:
better to use lv.get(0).isNull() since that 'may' change in the future (if more `null` behaving values there are).